web/controllers/metrics_controller.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Kastlex.MetricsController do
+  @moduledoc """
+    Entrypoint for Prometheus metrics collector.
+
+    Only topics with consumer groups are reported.
+  """
+  require Logger
+  use Kastlex.Web, :controller
+
+  def fetch(conn, _params) do
+    metrics = Enum.join(offsets(), "\n")
+    send_resp(conn, 200, metrics)
```

Comment:
I think we're missing content type here https://prometheus.io/docs/instrumenting/exposition_formats/#format-version-0-0-4
```
   conn
   |> put_resp_content_type("text/plain; version=0.0.4")
   |> send_resp(200, metrics)
```